game/entities.py

```
@@ -0,0 +1,252 @@
+                # Make the landing zone flat
+                for i in range(landing_zone_width):
+                    if start_pos + i < self.width:
+                        self.heights[start_pos + i] = zone_height
+                
+                # Add to landing zones list
+                self.landing_zones.append((start_pos, start_pos + landing_zone_width - 1))
+                
+                # Smooth transitions around landing zones
+                self._smooth_landing_zone_transitions(start_pos, start_pos + landing_zone_width - 1)
+    
+    def _smooth_landing_zone_transitions(self, start, end):
```

Ruff (0.8.2)

Missing return type annotation for private function `_smooth_landing_zone_transitions`

Add return type annotation: `None`

(ANN202)